front/src/applications/rollingStockEditor/views/RollingStockEditor.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useRef, useState } from 'react';
+
+import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
+import RollingStockCard from 'common/RollingStockSelector/RollingStockCard';
+import { useSelector } from 'react-redux';
+import { getRollingStockID } from 'reducers/osrdconf/selectors';
+import Loader from 'common/Loader';
+import { t } from 'i18next';
+import { sortBy } from 'lodash';
+import RollingStockEditorButtons from '../components/RollingStockEditorButtons';
+import RollingStockEditorForm from '../components/RollingStockEditorForm';
+
+export default function RollingStockEditor() {
+  const ref2scroll: React.RefObject<HTMLInputElement> = useRef<HTMLInputElement>(null);
+  const rollingStockID = useSelector(getRollingStockID);
+  const [isEditing, setIsEditing] = useState(false);
+
+  const [openedRollingStockCardId, setOpenedRollingStockCardId] = useState<number>();
+
+  const { rollingStocks } = osrdEditoastApi.useGetLightRollingStockQuery(
+    {
+      pageSize: 100,
+    },
+    {
+      selectFromResult: (response) => ({
+        ...response,
+        rollingStocks: sortBy(response.data?.results, ['metadata.reference', 'name']) || [],
+      }),
+    }
+  );
+  const getIndex = (id: number): number => rollingStocks.findIndex((rs) => rs.id === id);
+
+  return (
+    <div className="d-flex pt-5 mt-5">
+      <div className="d-flex ml-4 flex-column" style={{ width: '37%' }}>
+        {rollingStocks.length > 0 ? (
+          rollingStocks.map((data) => (
+            <div className="d-flex rollingstock-editor-list">
+              <div
+                role="button"
+                tabIndex={0}
+                className="align-self-start rollingstock-elements"
+                onClick={() => setIsEditing(false)}
+                style={{ width: '92%' }}
+              >
+                <RollingStockCard
+                  displayEditorButtons
+                  data={data}
+                  key={data.id}
+                  noCardSelected={openedRollingStockCardId === undefined}
+                  isOpen={data.id === openedRollingStockCardId}
+                  setOpenedRollingStockCardId={setOpenedRollingStockCardId}
+                  ref2scroll={rollingStockID === data.id ? ref2scroll : undefined}
+                />
+              </div>
+              <div
+                className={`align-self-start rollingstock-elements rollingstock-editor-buttons-group ${
+                  data.id === openedRollingStockCardId ? ' opened-rolling-stock-card' : ''
+                }`}
+              >
+                <RollingStockEditorButtons isCondensed setIsEditing={setIsEditing} />
+              </div>
+            </div>
```

Comment:
Display the buttons only if this card is selected
```suggestion
              { data.id === openedRollingStockCardId && <div
                className={'align-self-start rollingstock-elements rollingstock-editor-buttons-group'}
              >
                <RollingStockEditorButtons isCondensed setIsEditing={setIsEditing} />
              </div>
            </div>
```